albumentations/core/transforms_interface.py

```diff
@@ -141,6 +141,8 @@ def update_params(self, params, **kwargs):
             params["interpolation"] = self.interpolation
         if hasattr(self, "fill_value"):
             params["fill_value"] = self.fill_value
+        if hasattr(self, "mask_fill_value"):
+            params["mask_fill_value"] = self.mask_fill_value
```

Comment:
Are you sure that we need these changes? Looks like `CoarseDropout` can work with masks without these changes.

Reply:
There two ways to propagate params to apply functions - via "get_params" function in the class itself, or via this global "hack". For the sake of consistency with `fill_value` I thought to put it here. However we can discuss this decision :)

Reply:
Ok, I see about what you say. I think wee need some refactor and use same approach in all transforms, because today different transforms uses different methods.